src/Lib/Filters/ElasticSearch/SearchFilter.php

```diff
@@ -70,12 +83,48 @@ public function index(array $indexes)
      * @return string
      */
     public function __toString()
+    {
+        return '';
```

Comment:
Do we need this method with returning empty value instead of standard toString for object